hack/make-rules/make-help.sh

```diff
@@ -21,22 +21,25 @@ set -o pipefail
 readonly red=$(tput setaf 1)
 readonly reset=$(tput sgr0)
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/../..
 ALL_TARGETS=$(make -C "${KUBE_ROOT}" PRINT_HELP=y -rpn | sed -n -e '/^$/ { n ; /^[^ .#][^ ]*:/ { s/:.*$// ; p ; } ; }' | sort)
-CMD_TARGETS=$(ls -l "${KUBE_ROOT}/cmd" |awk '/^d/ {print $NF}')
+CMD_TARGETS=$(find . "${KUBE_ROOT}/cmd" |awk '/^d/ {print $NF}')
 CMD_FLAG=false
-PLUGIN_CMD_FLAG=false
+# export PLUGIN_CMD_FLAG
+#
+# .....
```

Comment:
@BenTheElder  
Here I chose to export, but I did not think how to write the comments, can you help me? thanks!

Reply:
why did we choose to export? the comment should explain what the value is and ideally what is expected to use it.